fgrpc/grpcrunner.go

```diff
@@ -116,17 +116,17 @@ func (grpcstate *GRPCRunnerResults) Run(t int) {
 type GRPCRunnerOptions struct {
 	periodic.RunnerOptions
 	Destination        string
-	Service            string        // Service to be checked when using grpc health check
-	Profiler           string        // file to save profiles to. defaults to no profiling
-	Payload            string        // Payload to be sent for grpc ping service
-	Streams            int           // number of streams. total go routines and data streams will be streams*numthreads.
-	Delay              time.Duration // Delay to be sent when using grpc ping service
-	ErrorPercent       int           // Error percentage
-	CACert             string        // Path to CA certificate for grpc TLS
-	CertOverride       string        // Override the cert virtual host of authority for testing
-	AllowInitialErrors bool          // whether initial errors don't cause an abort
-	UsePing            bool          // use our own Ping proto for grpc load instead of standard health check one.
-	UnixDomainSocket   string        // unix domain socket path to use for physical connection instead of Destination
+	Service            string // Service to be checked when using grpc health check
+	Profiler           string // file to save profiles to. defaults to no profiling
+	Payload            string // Payload to be sent for grpc ping service
+	Streams            int    // number of streams. total go routines and data streams will be streams*numthreads.
+	Delay              string // Delay to be sent when using grpc ping service
```

Comment:
I can see why you want to change this, but there should be a more efficient way to do.